app/src/main/java/com/jerboa/ui/components/inbox/InboxActivity.kt

```diff
@@ -290,13 +289,8 @@ fun InboxTabs(
                     )
 
                     val goToComment = { crv: CommentReplyView ->
-                        // Go to the parent comment or post instead for context
-                        val parent = getCommentParentId(crv.comment)
-                        if (parent != null) {
-                            appState.toComment(id = parent)
-                        } else {
-                            appState.toPost(id = crv.post.id)
-                        }
+                        // TODO navigate to comment (using appState.toComment()) once the route is supported
```

Comment:
What's blocking this?

Reply:
In my testing, navigating to the [Comment route](https://github.com/dessalines/jerboa/blob/main/app/src/main/java/com/jerboa/ui/components/common/Route.kt#L28) caused a crash, presumably because it is not supported by any existing composable :shrug: